### ISSUE TITLE
File Sync Issue

### ISSUE THREAD

**Issue by Sir-Kas** [jellyfin/jellyfin#12475]
### This issue respects the following points:

- [X] This is a **bug**, not a question or a configuration issue; Please visit our forum or chat rooms first to troubleshoot with volunteers, before creating a report. The links can be found [here](https://jellyfin.org/contact/).
- [X] This issue is **not** already reported on [GitHub](https://github.com/jellyfin/jellyfin/issues?q=is%3Aopen+is%3Aissue) _(I've searched it)_.
- [X] I'm using an up to date version of Jellyfin Server stable, unstable or master; We generally do not support previous older versions. If possible, please update to the latest version before opening an issue.
- [X] I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
- [X] This report addresses only a single issue; If you encounter multiple issues, kindly create separate reports for each one.

### Description of the bug

Jellyfin is not syncing the files/folder from the shared folders on my Synology NAS to the libraries on my Jellyfin server.

### Reproduction steps

1 - create a shared folder on Synology NAS.
2- - Download the jellyfin and sc-ffmpeg application on the Synology NAS.
3 - set the permissions on the shared folder that you created to read/write for both jellyfin and sc-ffmpeg.
4 - Open jellyfin, create a library, and link it to the shared folder on the Synology NAS. If you are creating a library for TV Shows make sure the files are named correctly in order to work with Jellyfins structure. 

### What is the current _bug_ behavior?

Jellyfin is not syncing the files/folder from the shared folders on my Synology NAS to the libraries on my Jellyfin server.

### What is the expected _correct_ behavior?

Once I upload files to the shared folder on the Synology NAS they should appear in the jellyfin library that I linked to the shared folder.

### Jellyfin Server version

Older*

### Specify version number

10.9.7

### Specify the build version

10.9.7

### Environment

```markdown
- OS: Synology OS
- Linux Kernel:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version: 6.0.1-3
- Playback Method:
- Hardware Acceleration:
- GPU Model:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
N/A
```

### Additional information

I made sure that the permissions where set correctly on all shared folders, and made sure to name the files correctly in order to work with Jellyfins' structure. Everything was working fine yesterday, now all of the sudden its like Jellyfin doesn't recognize my files when I upload them to the shared folder on my NAS. They aren't showing up in the jellyfin library.

**Comment by felix920506** [comment]
You are likely using a filesystem that doesn't support IONotify. You will have to wait for the rescan task for the new media to appear.

**Comment by Sir-Kas** [comment]
I have been chipping away at this media server everyday for the past 2 months and it has been working just fine, up until now. When you say wait for it to rescan you are talking about Jellyfin right? Jellyfin gives you the option to scan and update libraries within the GUI. If that’s what you are talking about, I’ve tried that and it’s still isn’t workingSent from my iPhoneOn Aug 22, 2024, at 11:46 AM, felix920506 ***@***.***> wrote:﻿
Closed #12475 as not planned.

**Comment by Sir-Kas** [comment]
Disregard that last message. Thanks for reach out. I appreciate your timeSent from my iPhoneOn Aug 22, 2024, at 2:29 PM, Rick Sanchez ***@***.***> wrote:﻿I have been chipping away at this media server everyday for the past 2 months and it has been working just fine, up until now. When you say wait for it to rescan you are talking about Jellyfin right? Jellyfin gives you the option to scan and update libraries within the GUI. If that’s what you are talking about, I’ve tried that and it’s still isn’t workingSent from my iPhoneOn Aug 22, 2024, at 11:46 AM, felix920506 ***@***.***> wrote:﻿
Closed #12475 as not planned.
